docs/config/cluster.rst

```diff
@@ -1086,8 +1086,7 @@ a node. There are several pools, but the important ones include:
 
 * ``write``: For index, update and delete operations, defaults to fixed
 * ``search``: For count/search operations, defaults to fixed
-* ``get``: For queries that are optimized to do a direct lookup by primary key,
-  defaults to fixed
+* ``get``: For queries on ``sys.shards`` and ``sys.nodes``. Defaults to fixed.
```

Comment:
For consistency with the above lines, maybe:

"For queries on ``sys.shards`` and ``sys.nodes``, defaults to fixed"